tests/console/rsync.pm

```diff
@@ -0,0 +1,61 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
```

Comment:
Feel free to keep just the current year (2018) unless you copied non-trivial code from the past.